src/main/java/com/manardenza/dao/HotelDaoImpl.java

```diff
@@ -18,6 +19,14 @@ public static HotelDaoImpl getInstance() {
         return instance;
     }
 
+    public ArrayList<Hotel> getHotelsByName(String hotelName) {
+        return null;
+    }
+
+    public ArrayList<Hotel> getHotelsByCity(String hotelCity) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1319)